modules/phase_field/include/materials/KKSXeVacSolidMaterial.h

```diff
@@ -15,7 +15,7 @@ class KKSXeVacSolidMaterial;
 template<>
 InputParameters validParams<KKSXeVacSolidMaterial>();
 
-class KKSXeVacSolidMaterial : public DerivativeFunctionMaterialBase
+class KKSXeVacSolidMaterial : public DerivativeFunctionMaterial
```

Comment:
Did you really just do a search replace `MaterialBase` -> `Material`?! Do you know about the word boundary marker `\b` for regular expressions?